cordapp-example/workflows-kotlin/src/main/kotlin/com/example/flow/ExampleFlow.kt

```diff
@@ -69,9 +71,14 @@ object ExampleFlow {
             progressTracker.currentStep = GENERATING_TRANSACTION
             // Generate an unsigned transaction.
             val iouState = IOUState(iouValue, serviceHub.myInfo.legalIdentities.first(), borrower)
+
+            // just printing some money, so I can payback later
+            val cashState = CashState(iouState.value.toLong(), iouState.borrower);
+
             val txCommand = Command(IOUContract.Commands.Create(), iouState.participants.map { it.owningKey })
             val txBuilder = TransactionBuilder(notary)
                     .addOutputState(iouState, IOUContract.ID)
+                    .addOutputState(cashState, CashContract.ID)
```

Comment:
You did not add the Create command so your contract will reject.

Reply:
Xavier is saying you need to add 2 commands (IOU create and Cash create); you added only one.